blackbelt/commands/sprint.py

```diff
@@ -0,0 +1,157 @@
+import click, re, datetime
+from blackbelt.config import config
+
+from blackbelt.handle_trello import get_api, verify, next_week, schedule_list
+
+@click.group(help='Team trello actions for sprints')
+def cli():
+    pass
+
+@cli.command(name='bp')
+def blueprint_plan():
+    """ Starting sprint planning for Team Blueprint™ """
+    api = get_api()
+
+    ##########################################
+
+    print "Retrieving the relevant columns ..."
+
+    product_columns = api.get_columns(config['trello']['product_board_id'])
+
+    current_sprint_column = None
+    next_sprint_column = None
+
+    for column in product_columns:
+        if re.match("^Sprint\\ (.*)\\ \\([0-9]+\\)$", column['name']):
```

Comment:
As the `r` before the string stands for *raw string*, the main benefit is that you don't need to use so much escaping inside of it. Highlighting in editors as regexp is just a convenient coincidence (raw strings are usually used for regexps). Thus, what you should do is:

```python
r'^Sprint\ (.*)\ \([0-9]+\)$'
```

because `r'^Sprint\\ (.*)\\ \\([0-9]+\\)$'` would result in `'^Sprint\\\\ (.*)\\\\ \\\\([0-9]+\\\\)$'`. And, because you do not need to escape spaces, you can even write:

```python
r'^Sprint (.*) \([0-9]+\)$'
```